app/Http/Controllers/Legacy/Web/CampaignsController.php

```diff
@@ -18,56 +19,63 @@ public function __construct()
         $this->middleware('role:admin,staff', ['only' => ['store', 'update', 'destroy']]);
```

Comment:
Re: Jen's comment, let's lock down this whole controller to authenticated staff/admin users (so normal users couldn't log in and view our campaign ID list).

Reply:
Great catch @ngjo ! Updated here and the documentation!